site/src/atomics/spacing.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Spacing Atomics
+
+Spacing Atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+## Class Pattern
+
+`<css-property-name>-<value>-<screensize>`
+
+For general information about the pattern, visit the [Atomics page](https://github.com/microsoft/atlas-design/blob/main/css/src/atomics/README.md).
+
+The following `<css-property-names>-<values>-<screensizes>` are supported.
+
+Accepted property names
+
+`margin, padding, margin-inline, padding-inline, margin-block, padding-block, margin-top, padding-top, margin-right, padding-right, margin-bottom, padding-bottom, margin-left, padding-left`
+
+Accepted values
+
+`none, xs, s, m, l, xl, xxl, xxxl`
+
+Accepted screensizes
+
+`tablet, desktop, widescreen`
+
+## Usages
+
+**Adding uniform spacing**
+
+Use `margin` with a size value to add spacing around an element.
```

Comment:
Use `margin` or `padding` with a size value to add uniform spacing around an element.

Reply:
(remove bolded line)